types/mapcmd/get.go

```diff
@@ -25,8 +25,7 @@ import (
 )
 
 const MapGetExample = `  # Get value of the given key from the map.
-  hzc map get -n mapname -k k1
-  hzc map get --key hello --name myMap
+  hzc map get --key hello --name myMap   # default key-type is string
```

Comment:
Do you think we can add the `key-type` here and get rid of the example below?

Reply:
How about just keeping the example below?